Misc/NEWS.d/next/Library/2022-05-25-00-21-28.gh-issue-91513.9VyCT4.rst

```diff
@@ -0,0 +1 @@
+Added ``%(taskName)s`` attribute to :mod:`logging` module for use with :mod:`asyncio` tasks.
```

Comment:
I think you could just say ``taskName`` here - is there a need for the format specifier? Anyway, that would depend on the style of the formatter used.

Reply:
Good point, percent style isn't the only format, will fix 🙂